test/core/transport/chttp2/hpack_encoder_test.cc

```diff
@@ -219,6 +221,152 @@ TEST(HpackEncoderTest, TestBasicHeaders) {
   delete g_compressor;
 }
 
+grpc_slice EncodeHeaderIntoBytes(
```

Comment:
This helper function mainly copies from https://github.com/grpc/grpc/blob/0536a75167cd4df5298f788df4b640c9f791c973/test/core/transport/chttp2/hpack_encoder_test.cc#L157-L158

I am not sure if I should reuse it or do what I have here.

Reply:
If we're introducing a new function that's largely the same, let's do the work to merge the two.